.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
is this file relevant to the upgrade and downgrade tests?

Reply:
Not. But, @mattmoor automatic style review is what kicked me to add this, to be sure next changes will be inline, as IDE be configured

Reply:
I was suggesting that we should perhaps follow the lead of the current `.gitignore` file and it leave editor specific configuration uncommited.  I would suggest dropping this editor-specific file

Reply:
Editor config is different, as it is cross IDE. It's supported nearly by all IDE and editors.

Reply:
I'd say it would be good to add this file to all Knative repositories through a different PR. But let's first create a GH issue because it would be good to know if this is desired by a broad community. The EditorConfig is not related to the upgrade/downgrade tests.

Reply:
Okay. I'll do that.

Reply:
Resolved by 52f603e